source/guides/dashboard/runs.md

```diff
@@ -22,6 +22,40 @@ Details of each run are displayed including:
 
 {% imgTag /img/dashboard/run-details.png "run-details" %}
 
+# Cancelling runs
+
+You can cancel a run currently in progress from the Dashboard. Runs can only be cancelled by members of a project.
+
+To cancel a run, click on a run in progress from the run list, then click on "Cancel Run" in the upper-right corner of the run details page.
+
+Cancelling a run in progress cannot be undone. Once you click "Cancel Run", a confirmation message will appear. Click the red "Yes, cancel this run" button to proceed.
+
+{% video local /img/snippets/cancelling-run.mp4 %}
+
+Once a run is cancelled:
+
+- The run status will update to cancelled
+- All incompleted instances of the run will be marked as cancelled
+- All test runners associated with the run will exit with an error
+- A message will appear on the run details page showing the time of cancellation and the user that made the request
+- The run will update to cancelled in GitHub if integration is enabled
+
+If you have Analytics enabled, cancelled runs will appear in the Runs over Time chart. Runs can also be filtered by cancellation status. 
+
+Cancelled runs do not count against your test recording limit.
```

Comment:
Oh, is this true? Even if 90% of the specs and tests completed running, and we only cancelled when the last spec was running, *none* of the test recordings count in their billing?

Reply:
That's not accurate, no. We count all tests recorded before the run was canceled. If they ran 90% of their test suite, then canceled, that first 90% would still be included in their usage.

Reply:
Yes, I meant the tests that hadn't run yet. I'll definitely clarify that.

Reply:
I also confirmed that when specs have already run and recorded, the test results are available in the run details. I'll add that info as well.